tox.ini

```diff
@@ -7,39 +7,24 @@ envlist =
   py27-lint-readme,
   py27-pydocstyle
 
-skipsdist = True
 skip_missing_interpreters = True
 
-[travis]
-python =
-  2.7: py27
-  3.4: py34
-  3.5: py35
-  3.6: py36
-  3.7: py37
-
 [testenv]
-passenv =
-  CI
-  TRAVIS
-  TRAVIS_*
+passenv = CI TRAVIS*
```

Comment:
This shouldn't be necessary to make tox-travis work. I don't think it should cause issues either, though.

Reply:
It's necessary to be able to upload coverage reports to codecov, though